src/geovista/examples/vectors.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""Importable and runnable geovista example.
+
+Notes
+-----
+.. versionadded:: 0.4.0
+
+"""
+from __future__ import annotations
+
+import numpy as np
+import pyvista as pv
+
+import geovista
+import geovista as gv
+
+
+def main() -> None:
+    """Create vectors plotting inspired by cartopy."""
+    # make cool swirly pattern
+    sphere = pv.Sphere(gv.common.RADIUS)
+    vectors = np.vstack(
+        (
+            -sphere.points[:, 1],
+            sphere.points[:, 0],
+            sphere.points[:, 2] * 0.0,
+        )
+    ).T
+    sphere["vectors"] = vectors * 0.1
+    sphere.set_active_vectors("vectors")
+
+    # plot mesh
+    plotter = gv.GeoPlotter()
+    plotter.set_background("gray")
+    plotter.enable_anti_aliasing("ssaa")
```

Comment:
@tkoyama010 This anti-aliasing option really makes an amazing difference.

How about you push a separate PR to make this the default anti-aliasing option for the `geovista.geoplotter.GeoPlotterBase` ?

Reply:
MEMO: after offline discussion, it was decided to add a FAQ to the Discussion to promote its use by users.

Reply:
@tkoyama010 Actually, after reading up further to understand the different anti-aliasing options and the pros vs cons ([PyVista - Anti-Alisasing](https://docs.pyvista.org/version/stable/examples/02-plot/anti-aliasing.html#anti-aliasing)), I've come to the conclusion that `ssaa` as a default option might be detrimental to most users on the grounds of the performance hit.

I think it would be better to maintain the `pyvista` anti-aliasing default, but allow users to explicitly opt-in to another strategy buy manually configuring it themselves; as only they know their GPU rendering capabilities available.

However, I think it's fine to include the `enable_anit_aliasing("ssaa")` configuration here in this example :+1: